### PR TITLE
sync-diff-inspector: add annontation for option configuration items

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -80,24 +80,27 @@ check-struct-only = false
 
 ######################### Datasource config #########################
 [data-sources]
-[data-sources.mysql1] # mysql1 is the only ID for the database instance. It is used for following `task.source-instances/task.target-instance` configuration.
+[data-sources.mysql1] # mysql1 is the only custom ID for the database instance. It is used for following `task.source-instances/task.target-instance` configuration.
```
Comment (hfxsd) [inline sync-diff-inspector/sync-diff-inspector-overview.md:83]:
```suggestion
[data-sources.mysql1] # mysql1 is the only custom ID for the database instance. It is used for the following `task.source-instances/task.target-instance` configuration.
```

**Review by Leavrth — APPROVED** [review]

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 2]
```diff
@@ -57,9 +57,9 @@ The configuration of sync-diff-inspector consists of the following parts:
 
 - `Global config`: General configurations, including number of threads to check, whether to export SQL statement to fix inconsistent tables, whether to campare the data and so on.
 - `Databases config`: Configures the instances of the upstream and downstream databases.
-- `Tables config`: Special configurations for specific tables, including specified ranges, columns to be ignored and so on (optional).
-- `Routes`: Rules for upstream multiple schema names to match downstream single schema names (optional).
+- `Routes`: Rules for upstream multiple schema names to match downstream single schema names **(optional)**.
 - `Task config`: Configures the tables for checking. If some tables have a certain mapping relationship between the upstream and downstream databases or have some special requirements, you must configure these tables.
+- `Table config`: Special configurations for specific tables, including specified ranges, columns to be ignored and so on **(optional)**.
```
Comment (hfxsd) [inline sync-diff-inspector/sync-diff-inspector-overview.md:62]:
```suggestion
- `Table config`: Special configurations for specific tables, such as specified ranges and columns to be ignored **(optional)**.
```
including一般用于穷举。这里用such as比较好，而且用such as后面也不能跟 and so on

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Pull request merged**